### DISCUSSION TITLE
Is it possible to configure musicbrainz to only write specific fields to the database?

### DISCUSSION THREAD

**Discussion by arsaboo** [beetbox/beets#4483] [Q&A]
Now that we have a few tag sources (MB, Spotify, Discogs) and more that will be added in the future, I was wondering if there is a way to configure the MusicBrainz plugin to write only certain fields. 

For example, I am using the Spotify plugin to import metadata/tags and want to continue to use that. However, given that Beets is designed around MusicBrainz and many plugins (e.g., `missing` relies on MB ids to work), I would also like to import the MB ids. The only was this is currently possible is to do multiple imports - first with the Spotify plugin and get all the data from Spotify and then another MB plugin to get the MB data. The problem, however, is that the MB plugin overwrites all the previously imported data. 

Hence, my question (and maybe a feature suggestion) - is there a way to configure the fields to be imported by the MB plugin like several other plugins offer? For example, I would only like to import only fields that have the `mb_` subscripts that are used in other plugins.

**Comment by JOJ0** [comment]
https://github.com/beetbox/beets/issues/4087#issuecomment-1250003965

related feature request, including a proposal how the feature could look like but it speaks about global configuration of which fields to populate.

Your idea here is about a plugin-wise configuration!
